app/views/users/edit.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="d-flex align-items-left justify-content-left mt-4 ml-4">
+  <h1>Edit User</h1>
+</div>
+
+<%# <form action="/users" method="Post" accept-charset="UTF-8">
```

Comment:
- [ ] It so much better if you use `users_path` helper for action instead of  `/users` i.g `action="users_path"` Please change this line of code. 🥇 👏